UNRELEASED.md

```diff
@@ -7,6 +7,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Enhancements
 
 - `TextField` no longer uses `componentWillReceiveProps`([#628](https://github.com/Shopify/polaris-react/pull/628))
+- `TextField` uses autocomplete="nope" when `autoComplete` prop is `false` ([#708](https://github.com/Shopify/polaris-react/pull/708))
```

Comment:
Can you change this to use past tense instead of present tense (`Fixed`) and move it into the `Bug fixes` section, as this feels more like a bug fix than an enhancement?

Reply:
Done